distribution/tools/keystore-cli/src/test/java/org/elasticsearch/bootstrap/BootstrapTests.java

```diff
@@ -60,7 +61,9 @@ public void testLoadSecureSettings() throws Exception {
             keyStoreWrapper.save(configPath, new char[0]);
         }
         assertTrue(Files.exists(configPath.resolve("elasticsearch.keystore")));
-        try (SecureSettings secureSettings = Bootstrap.loadSecureSettings(env)) {
+        MockTerminal mockTerminal = new MockTerminal();
+        mockTerminal.addSecretInput("");
```

Comment:
I've added a check so that (in java) we don't try the potentially blocking read from standard input unless we know that the keystore exists and is password protected.